app/questionnaire/dependencies.py

```diff
@@ -15,35 +15,42 @@
     from app.questionnaire.path_finder import PathFinder  # pragma: no cover
 
 
-def get_block_ids_for_calculated_summary_dependencies(
+def get_block_ids_for_dependencies(
```

Comment:
Can we force named arguments here? It's too many of them to follow.

Reply:
Forced 👍